tests/test_json.py

```diff
@@ -166,6 +166,8 @@ def test_arrindex(client):
     client.json().set("arr", Path.root_path(), [0, 1, 2, 3, 4])
     assert 1 == client.json().arrindex("arr", Path.root_path(), 1)
     assert -1 == client.json().arrindex("arr", Path.root_path(), 1, 2)
+    assert 4 == client.json().arrindex("arr", Path.root_path(), 4)
+    # assert -1 == client.json().arrindex("arr", Path.root_path(), 4, start=0, stop=-1)
```

Comment:
Why is this line commented out?

Reply:
Glad you caught this - check was failing due to a logic bug in `json().arrindex()`. Fixed and uncommented test.